x-pack/test/security_solution_endpoint/apps/endpoint/artifact_entries_list.ts

```diff
@@ -114,39 +266,23 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
           // Checks if fleet artifact has been updated correctly
           await checkFleetArtifacts(
```

Comment:
I know it's not the scope of this PR, but consider refactoring arguments to be an object. Passing this many arguments may be unwieldy.